test/case/infix_containers/container_phys/test.py

```diff
@@ -74,7 +72,26 @@
         with test.step("Verify host:data can ping 10.0.0.2"):
             ns.must_reach(DUTIP)
 
-        with test.step("Verify container 'web-phys' is reachable on http://10.0.0.2:91"):
+        with test.step("Verify container is reachable on http://10.0.0.2:91"):
             until(lambda: url.nscheck(ns, "It works"), attempts=10)
 
+        with test.step("Verify modifying a running container takes"):
+            # Regression test for issue #930
+            data = to_binary(BODY)
+
+            target.put_config_dict("infix-containers", {
+                "containers": {
+                    "container": [{
+                        "name": f"{NAME}",
+                        "mount": [{
+                            "name": "index.html",
+                            "content": f"{data}",
+                            "target": "/var/www/index.html"
+                        }]
+                    }]
+                }
+            })
+
+            until(lambda: url.nscheck(ns, MESG), attempts=10)
+
```

Comment:
```suggestion
        with test.step("Add a content mount, overriding index.html"):
            # Regression test for issue #930
            data = to_binary(BODY)

            target.put_config_dict("infix-containers", {
                "containers": {
                    "container": [{
                        "name": f"{NAME}",
                        "mount": [{
                            "name": "index.html",
                            "content": f"{data}",
                            "target": "/var/www/index.html"
                        }]
                    }]
                }
            })

        with test.step("Verify that index.html has been updated with the new content"):
            until(lambda: url.nscheck(ns, MESG), attempts=10)

```

From the perspective of someone trying to reproduce this test, I think it is better to break this up into two steps, and to provide some hint about what is being done.

Reply:
Good point!